Tests/LibWeb/Text/input/HTML/cross-origin-window-properties.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<script src="../include.js"></script>
+<iframe name="testFrame"></iframe>
+<script>    
+    asyncTest(done => {
+        const frame = document.querySelector("iframe");
+        // Taken from: https://html.spec.whatwg.org/multipage/nav-history-apis.html#crossoriginproperties-(-o-)
+        const crossOriginWindowProperties = [
+            "window",
+            "self",
+            "location",
+            "close",
+            "closed",
+            "focus",
+            "blur",
+            "frames",
+            "length",
+            "top",
+            "opener",
+            "parent",
+            "postMessage",
+        ];
+        frame.onload = () => {
+            for (const property of crossOriginWindowProperties) {
+                const value = frame.contentWindow[property];
+            }
+            println("PASS (didn't crash)");
+            done();
+        }
+    
+        window.open("about:newtab", "testFrame");
```

Comment:
Using `about:newtab` here feels a bit fragile, but it was the only way I could find to load a cross origin url with local files.
Using a `http://` or `https://` url also works, but obviously isn't great for a test. 
We currently treat all `file://` urls as being from the same origin, although it isn't clear from the spec whether that is correct or not.
`about:srcdoc` and `about:blank` also don't exercise the relevant code path.

Reply:
It's a bit hackish yeah, but I don't have a better suggestion unless we want to start running a web server as part of the test system.
We can come back to this if we still care, later on :)

Reply:
I've added a comment here to explain why I've done this and that a better solution should be used if it becomes available.